src/Page/NotAlone.elm

```diff
@@ -267,6 +268,16 @@ quoteStyle =
         ]
 
 
+relatableQuoteStyle : Style
```

Comment:
Not sure.. is this for emphasis? I don't know how much a 0.1rem bump will look intentional

Reply:
![image](https://user-images.githubusercontent.com/861172/87532150-9ffa7800-c68a-11ea-8199-8ec8d4a418f1.png)


Looks like this - it's subtle, but I think better than them not being different. I didn't want to mess with the design too much or make it seem like that one was more important. Just a clue to keep your place visually when it "opens". We will have to test. I guess if it confuses the first person too much, you can come up with a different visual clue? Like a line ?

It's hard to make "design" decisions without a design yet. I'm trying to stick as close as possible to layout and typography only at this stage.

Reply:
I think the effect is quite subtle either way, but emphasises what was there before, rather than what's new.. would it be better to emphasise the new content? Something like:
![image](https://user-images.githubusercontent.com/32434620/87561004-fc25c200-c6b3-11ea-991f-744ce1fe83ce.png)

Reply:
Yes, that makes more sense. Do that.

It also helps with the reassuring text not seeming to be part of the story.